clojure_liberator/src/clojure_liberator/core.clj

```diff
@@ -21,6 +21,20 @@
    :type       type-name
    :attributes (dissoc value :id)})
 
+(defn- update-vals [map vals f]
+  (reduce #(update-in %1 [%2] f) map vals))
+
+(defn- parse-date [value]
```

Comment:
Me, too. This is something that I still need to understand better. There's
a library called Component that I want to look into for that.